src/uipath/models/attachment.py

```diff
@@ -0,0 +1,31 @@
+from datetime import datetime
+import uuid
+from typing import Any, Dict, List, Optional
+
+from pydantic import BaseModel, ConfigDict, Field
+
+
+class Attachment(BaseModel):
+    """Model representing an attachment in UiPath.
+
+    Attachments can be associated with different objects in UiPath such as
+    actions, tasks, or processes, and contain binary files or documents.
+    """
+
+    model_config = ConfigDict(
+        validate_by_name=True,
+        validate_by_alias=True,
+        use_enum_values=True,
+        arbitrary_types_allowed=True,
+        extra="allow",
+        json_encoders={datetime: lambda v: v.isoformat() if v else None},
+    )
+
+    name: str = Field(alias="Name")
+    organization_unit_id: int = Field(alias="OrganizationUnitId")
+    tenant_id: int = Field(alias="TenantId")
```

Comment:
we should only work with GUIDs (folder key, tenant key, etc) - if those aren't available, we can just remove these